sources/api/openapi.yaml

```
@@ -324,3 +324,79 @@ paths:
           description: "Reboot requested"
         500:
           description: "Server error"
+
+  /actions/refresh-update:
+    post:
+      summary: "Query update repository and refresh list of updates"
+      operationId: "refresh_update"
+      responses:
+        204:
+          description: "Successful request"
+        500:
+          description: "Server error"
+        423:
+          description: "Update lock held. Try again in a moment"
```

```suggestion
          description: "Update write lock held. Try again in a moment"
```
Should only happen if there's an exclusive lock, might help to clarify.  Same below.